views/partials/featured_breach.hbs

```diff
@@ -0,0 +1,15 @@
+{{#if featuredBreach }}
+  <div class="account">
+    <div id="breach-title-wrapper">
+      <div class="image-wrap">
+        <img alt="{{ featuredBreach.Name }}" src="img/logos/{{ featuredBreach.Name }}.{{ featuredBreach.LogoType }}">
+      </div>
+      <span class="breach-headline">{{ featuredBreach.Name }} Breach</span>
```

Comment:
Should this be `.Name` or `.Title`?

Actually, looking at ./views/partials/breach_listing.hbs (from a few files up), it seems like we do have a few inconsistencies betweenst that partial and this partial:

https://github.com/mozilla/blurts-server/blob/ea4947b4f4a0a5237f1e43267659a5565fad8227/views/partials/breach_listing.hbs#L1-L11

Note that we have stuff like `<span class="medium">Breach date:</span>` vs `<span class="bold">Breach Date</span>`:

```html
<span class="breach-info"><span class="medium">Breach date:</span> {{prettyDate BreachDate }}</span>
```

vs 

```html
<span class="breach-info"><span class="bold">Breach Date</span> {{prettyDate featuredBreach.BreachDate }}</span>
```